docs/background/plot_00_conceptual_intro.py

```diff
@@ -200,7 +200,9 @@
 #     (4). This method is used under the hood during optimization.
 
 # %%
-### More general GLMs
-# So far, we have focused on the relatively simple LNP model of spike generation, which is a special case of a GLM. The LNP model has some known shortcomings[^1]. For instance, LNP ignores things like refactory periods and other history-dependent features of spiking in a neuron. As we will show in other demos, such _spike history filters_ can be built into GLMs to give more accurate results. We will also show how, if you have recordings from a large _population_ of neurons simultaneously, you can build connections between the neurons into the GLM in the form of _coupling filters_. This can help answer the degree to which activity is driven primarily by the input X, or by network influences in the population.
+# ## More general GLMs
+# So far, we have focused on the relatively simple LNP model of spike generation, which is a special case of a GLM. The LNP model has some known shortcomings[$^{[1]}$](#ref-1). For instance, LNP ignores things like refactory periods and other history-dependent features of spiking in a neuron. As we will show in other demos, such _spike history filters_ can be built into GLMs to give more accurate results. We will also show how, if you have recordings from a large _population_ of neurons simultaneously, you can build connections between the neurons into the GLM in the form of _coupling filters_. This can help answer the degree to which activity is driven primarily by the input X, or by network influences in the population.
```

Comment:
These footnotes were rendering correctly (https://nemos.readthedocs.io/en/latest/generated/background/plot_00_conceptual_intro/#fnref:1). Do you still want to change how they're done?

Reply:
they weren't rendered correctly anymore. I simplified the mkdocs.yaml but even with only the footnotes extension it would not render. 
Anyways, I like the reference section more.